### ISSUE TITLE
broken example

### ISSUE THREAD

**Issue by alexander-akait** [postcss/postcss-calc#117]
```css
a {
  width: calc(min(max(var(--foo), 0), 100))
}
```

Ref: https://github.com/cssnano/cssnano/issues/949

**Comment by alexander-akait** [comment]
@ludofischer We should support `min`/`max`/etc as top function too, i.e. `width: max(100px, 200px)`

**Comment by ludofischer** [comment]
@alexander-akait Do you mean `max(100px, 200px) -> 200px`. That would be an additional feature, right? I don't think it solves the current bug.

**Comment by alexander-akait** [comment]
Yes, it is feature, now we have more math in CSS, so will be great to support it too

**Comment by ludofischer** [comment]
Solved in new parser.
